content/sensu-go/6.1/observability-pipeline/observe-events/events.md

```diff
@@ -120,7 +120,32 @@ Sensu events contain:
 ## Use event data
 
 Observability data in events is a powerful tool for automating monitoring workflows.
-For example, you can [reduce alert fatigue][23] by filtering events based on the event `occurrences` attribute.
+For example, you can use the [`state` attribute][36] to provide handlers with more information about changes in check status and [reduce alert fatigue][23] by filtering events based on the event `occurrences` attribute.
```

Comment:
```suggestion
For example, the [`state` attribute][36] is useful to handlers, providing high-level description of check status. This can help to [reduce alert fatigue][23] by filtering events based on this attribute.
```